tests/test_routes.py

```diff
@@ -112,6 +112,39 @@ def test_get_shopcart_not_found(self):
         """It should not Read a Shopcart that is not found"""
         response = self.client.get(f"{BASE_URL}/0")
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+    
+    def test_get_item(self):
+        """It should Read an item from a shopcart"""
+        test_shopcart = self._create_an_empty_shopcart(1)[0]
+        item = ItemFactory()
+        response = self.client.post(
+            f"{BASE_URL}/{test_shopcart.id}/items",
+            json=item.serialize(),
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        data = response.get_json()
+        logging.debug(data)
+        item_id = data["id"]
+
+        response = self.client.get(
+            f"{BASE_URL}/{test_shopcart.id}/items/{item_id}",
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.get_json()
+        logging.debug(data)
+        self.assertEqual(data["shopcart_id"], test_shopcart.id)
+        self.assertEqual(data["quantity"],item.quantity)
+        self.assertEqual(data["price"],item.price)
+
```

Comment:
we would need one more test case for getting an item from a non-existent `shopcart_id`